Snakefile

```diff
@@ -303,11 +308,18 @@ if config["US"].get("retrieve_US_databundle", True):
                 "analysis/gdrive_data/data/electricity_demand_data/{filename}",
                 filename=[
                     "use_es_capita.xlsx",
-                    "EIA930_2021_Jan_Jun_opt.csv",
-                    "EIA930_2021_Jul_Dec_opt.csv",
                     "HS861 2010-.xlsx",
+                    "demand_data/table_10_EIA_utility_sales.xlsx",
+                    "demand_data/Electric_Retail_Service_Territories.geojson",
                 ],
+            ), 
+            expand(
+                pathlib.Path("analysis","gdrive_data","data","electricity_demand_data","EIA930_{demand_year}_Jan_Jun_opt.csv"), **config["US"], 
+            ), 
+            expand(
+                pathlib.Path("analysis","gdrive_data","data","electricity_demand_data","EIA930_{demand_year}_Jul_Dec_opt.csv"), **config["US"], 
```

Comment:
It looks like there is an inconsistency there as `expand()` is applied over strings in the other parts of Snakemake